evm/src/cpu/kernel/tests/balance.rs

```diff
@@ -0,0 +1,118 @@
+use anyhow::Result;
+use eth_trie_utils::partial_trie::PartialTrie;
+use ethereum_types::{Address, BigEndianHash, H256, U256};
+use keccak_hash::keccak;
+use rand::{thread_rng, Rng};
+
+use crate::cpu::kernel::aggregator::KERNEL;
+use crate::cpu::kernel::constants::global_metadata::GlobalMetadata;
+use crate::cpu::kernel::interpreter::Interpreter;
+use crate::cpu::kernel::tests::mpt::nibbles_64;
+use crate::generation::mpt::{all_mpt_prover_inputs_reversed, AccountRlp};
+
+// Test account with a given code hash.
+fn test_account(balance: U256) -> AccountRlp {
+    AccountRlp {
+        nonce: U256::from(1111),
+        balance,
+        storage_root: PartialTrie::Empty.calc_hash(),
+        code_hash: H256::from_uint(&U256::from(8888)),
+    }
+}
+
+// Stolen from `tests/mpt/insert.rs`
+// Prepare the interpreter by inserting the account in the state trie.
+fn prepare_interpreter(
+    interpreter: &mut Interpreter,
+    address: Address,
+    account: &AccountRlp,
+) -> Result<()> {
```

Comment:
This is the same function as in #796. We can make it public to avoid redundancy.